ray-requirements.txt

```diff
@@ -2,3 +2,4 @@ ray==2.2.0
 pandas
 scikit-learn==1.0.2
 SQLAlchemy==1.3.24
+pyarrow==10.0.1
```

Comment:
not seeing this used. is it an internal ray requirement?

Reply:
Yes, the test won't run without it and Ray doesnt have it as a dependency for some reason. https://github.com/ray-project/ray/issues/25952

Reply:
The recommendation in this thread you quoted is to do `pip install "ray[data]"` rather than installing pyarrow etc., other libs might be missing too. best if we build our docker image with ray[data].

Reply:
will change